<?xml version="1.0"?>
<launch>

  <node pkg="ublox_gps" type="ublox_gps" name="puma_ublox" output="screen" clear_params="true" respawn="true" respawn_delay="5">
    <rosparam command="load" file="$(find puma_bringup_jetson)/config/zed_f9p.yaml"/>
    <remap from="puma_ublox/fix"           to="puma/sensors/gps/fix"/>
    <remap from="puma_ublox/fix_velocity"  to="puma/sensors/gps/fix_velocity"/>
  </node>

</launch> 